test/Intent/Runner.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Intent;
+
+using System.Reflection;
+
+public class Runner
+{
+    public static void Run(IEnumerable<string> path, IRunLogger logger)
+    {
+        foreach (var p in path)
```

Comment:
Maybe `path` instead of `paths`, or `currentPath`.

Reply:
done, used path